dts/bindings/mipi-dbi/zephyr,mipi-dbi-spi.yaml

```diff
@@ -28,20 +28,16 @@ properties:
       Reset GPIO pin. Set high to reset the display
 
   xfr-min-bits:
-    type: int
-    default: 8
+    type: string
+    default: "MIPI_DBI_SPI_XFR_8BIT"
     description:
       On rare types of SPI interfaces, discrete shift registers can be found
       whose task is to convert the serial SPI bit stream to the parallel MCU
       interface with clock and bit accuracy. Typically, these are 16 bits wide.
-
-      Use the macros, not the actual enum value. Here is the concordance list
-      (see dt-bindings/mipi_dbi/mipi_dbi.h)
-         8     MIPI_DBI_SPI_XFR_8BIT
-        16     MIPI_DBI_SPI_XFR_16BIT
+      These definitions should match those in dt-bindings/mipi_dbi/mipi_dbi.h
     enum:
-      - 8
-      - 16
+      - "MIPI_DBI_SPI_XFR_8BIT"
```

Comment:
In the past I've added migration guide entries for changes like these: https://github.com/zephyrproject-rtos/zephyr/pull/81293/files#diff-916ae40f1edf47d192b2d78a82eaf71c6352ff1bd3c679654c3e31a6a5f31930R58.

This is a nonblocking suggestion though, because I don't expect this property is being changed from the default very often

Reply:
I see and for the old (long time stable) DTS property `mipi-mode` that is really cool and helpful. I maybe wrong, but the `xfr-min-bits` property is extremely new, was merged in after v4.0 release tag and so it was never part of the v4.0 release. That's why I've skipped the notice in the migration guide from v4.0 to v4.1.